tests/serverpod_test_server/test_integration/column_operations/column_uuid_test.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+
+class ColumnUuidEndpoint extends Endpoint {
```

Comment:
Question: This looks strange as well, should probably be removed right? No endpoint is needed for these tests now.